caf-audit-service-container/src/integration-test/java/com/hpe/caf/services/audit/api/WebserviceClientAuditIT.java

```diff
@@ -133,7 +133,7 @@ public void randomiseTenantId() {
     @AfterMethod
     public void cleanUp() throws AuditConfigurationException {
         RestHighLevelClient client
-                     = ElasticAuditRestHighLevelClientFactory.getHighLevelClient(ES_HOSTNAME_AND_PORT);
+                     = ElasticAuditRestHighLevelClientFactory.getHighLevelClient(ES_HOSTNAME_AND_PORT, String.valueOf(ES_PORT));
```

Comment:
I don't understand that these methods are sending the port twice now.  Doesn't it seem wrong to you?

Reply:
Fixed by removing extra port parameter to getHighLevelClient method.